DigitalLearningSolutions.Web/Controllers/TrackingSystem/Centre/Administrator/AdministratorController.cs

```diff
@@ -144,26 +145,39 @@ public IActionResult UnlockAccount(int adminId)
         [Route("{adminId:int}/DeactivateAdmin")]
         [HttpGet]
         [ServiceFilter(typeof(VerifyAdminUserCanAccessAdminUser))]
-        public IActionResult DeactivateAdmin(int adminId, int? returnPage)
+        public IActionResult DeactivateOrDeleteAdmin(int adminId, int? returnPage)

```

Comment:
Looks like the action name specified in the form has not been updated to match this, so we start getting problems when trying to submit it.